src/components/DayPickerRangeController.jsx

```diff
@@ -160,6 +160,8 @@ export default class DayPickerRangeController extends React.Component {
     if (focusedInput !== END_DATE) return false;
 
     if (startDate) {
+      if (minimumNights === 1) return isSameDay(day, startDate);
```

Comment:
Oh you're right. I just tested this with my use case, which was without a minimumNights prop (so default 1). Will look into a more robust fix